### ISSUE TITLE
[Bug Report]: Contributing link to module proposal is broke

### ISSUE THREAD

**Issue by wsmelton** [Azure/bicep-registry-modules#210]
### Module path

n/a

### Bicep version

n/a

### Describe the bug

The contributing guide has a link that should point to the [issue template](https://github.com/Azure/bicep-registry-modules/issues/new?labels=Module+Proposal&template=module_proposal.yml&title=%5BModule+Proposal%5D%3A) for a module proposal but sends you to an empty new issue template.

### To reproduce

Go to [Contributing guide](https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md)
Click on issue template link under section `Making a Proposal`

### Code snippet

```Bicep
### Making a proposal

<!-- TODO: update the issue link once the repo is public -->

Before creating a new module, you must fill out this [issue template](https://github.com/Azure/bicep-registry-modules/issues/new) to make a proposal. Each module needs to have its own proposal. Please avoid including multiple modules in one issue. Once the proposal is approved, proceed with the following steps. You should not send out a pull request to add a module without an associated approval as the pull request will be rejected.
```

### Relevant log output

```Shell
n/a
```
